msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Designing your meeting extension
 author: heath-hamilton
-description: Learn how to design apps in Teams meetings and get the Microsoft Teams UI Kit, in-meeting tab and use cases, responsive behavior and shared meeting stage, and theme and navigation.
+description: Learn how to design apps in Teams meetings and get the Microsoft Teams UI Kit, in-meeting tab, use cases, responsive behavior, shared meeting stage, theme, and navigation.
```

Comment:
The first sentence reads something like, Learn how to get UI kit. Needs an edit.

Complete sentence also doesn't read well. Second half doesn't flow with the first half of the sentence.

Reply:
@ashishguptaiitb,

Addressed this in https://github.com/MicrosoftDocs/msteams-docs/pull/6385